model_compression_toolkit/core/keras/pruning/prune_keras_node.py

```diff
@@ -0,0 +1,140 @@
+import keras.layers
+import numpy as np
+
+from model_compression_toolkit.core.common.framework_info import FrameworkInfo
+from model_compression_toolkit.core.common import BaseNode
+
+
+
+
+
+def prune_keras_entry_node(node: BaseNode,
```

Comment:
same, what is the framework-specific implementation file if not for such functions?